internal/handlers/pg/msg_listdatabases.go

```diff
@@ -67,7 +67,7 @@ func (h *Handler) MsgListDatabases(ctx context.Context, msg *wire.OpMsg) (*wire.
 			for _, name := range tables {
 				var tableSize int64
 				fullName := pgx.Identifier{databaseName, name}.Sanitize()
-				err = tx.QueryRow(ctx, "SELECT pg_total_relation_size($1)", fullName).Scan(&tableSize)
+				err = tx.QueryRow(ctx, "SELECT COALESCE(pg_total_relation_size($1), 0)", fullName).Scan(&tableSize)
```

Comment:
There should be explaining why we use `COALESCE`.

If the reason is "table was there when we called `Tables(), but was removed in the meantime` – maybe we should do the same as in the Tigris handler? https://github.com/FerretDB/FerretDB/blob/939bfcba92cec566e89de8dccfe2e46ac6dd0a88/internal/handlers/tigris/msg_listdatabases.go#L57-L60